requirements.txt

```diff
@@ -35,3 +35,4 @@ virtualenv==15.1.0
 webencodings==0.5.1
 websocket-client==0.44.0
 wrapt==1.10.10
+configparser==3.5.0
```

Comment:
why add a third-party lib when python has this available in all the versions we use?

Reply:
Indeed, was my mistake. The IDE kept saying it wasn't installed and I thought it was needed for Python 2. 
Will remove in future commit then.